### ISSUE TITLE
Example code in "too-many-ancestors /R0901" documentation incorrectly has platypus as non-egg laying mammals. 

### ISSUE THREAD

**Issue by mayafox** [PyCQA/pylint#9675]
### Bug description

```python
https://pylint.readthedocs.io/en/latest/user_guide/messages/refactor/too-many-ancestors.html <<

class Playtypus(Mammal):
    beaver_tailed = True
    can_swim = True
    has_beak = True
    lays_egg = False  <<<< should be set to True
    protected_specie = True
    venomous = True
```

https://en.wikipedia.org/wiki/Platypus
```


### Configuration

_No response_

### Command used

```shell
Documentation issue.
```

### Pylint output

```python
None, Documentation issue
```

### Expected behavior

Correct example in the documentation
class Playtypus(Mammal):
    beaver_tailed = True
    can_swim = True
    has_beak = True
    lays_egg = True
    protected_specie = True
    venomous = True

### Pylint version

```shell
pylint 3.3.0
```

**Comment by Pierre-Sassoulas** [comment]
I really like this bug :D do you want to fix it ? The source is in Doc/data/message/t/too-many...
